tests/test_dle.py

```diff
@@ -216,6 +217,26 @@ def test_cache_blocking_structure_optrelax_customdim():
                      'd,x0_blk0,y0_blk0,z0_blk0,x,y,z')
 
 
+def test_cache_blocking_structure_optrelax_defaultdim_alone():
+    grid = Grid(shape=(8, 8, 8))
+    d = DefaultDimension(name='d', default_value=2)
+    time = grid.time_dim
+    x, y, z = grid.dimensions
+
+    u = TimeFunction(name="u", grid=grid)
+    f = Function(name="f", grid=grid, dimensions=(d, x, y, z),
+                 shape=(2,) + grid.shape)
+
+    eqn = Inc(f, u*cos(time*d))
+
+    op = Operator(eqn, opt=('advanced', {'blockrelax': 'device-aware'}))
+
+    _, _ = assert_blocking(op, {'d0_blk0', 'x0_blk0'})
```

Comment:
Why do you need to create dummy variables here? Could you not just have `assert_blocking(op, {'d0_blk0', 'x0_blk0'})` on its own?

Reply:
copy-paste leftover